roles/branch-validation/tasks/main.yml

```diff
@@ -68,17 +61,14 @@
     nics:
       - net-name: internal
     name: "validate-{{ item }}"
+    availability_zone: "nova:{{ item }}"
```

Comment:
this use of `nova` is not going to work with new 2.1.x stacks, as the AZ will be based on the node's type.

That said, when you're looping over `groups['compute']`, you have access to `hostvars[item]['compute_ag']` which will give you the name of the AZ (we expose the AG as an AZ). You can default that variable grab to just `|default('nova')` to cover the cases where it isn't defined:

`availability_zone: "{{ hostvars[item]['compute_ag']|default('nova') }}:{{ item }}"`